src/collection-view.js

```diff
@@ -101,7 +101,7 @@ Marionette.CollectionView = Marionette.View.extend({
   _onCollectionAdd: function(child, collection, opts) {
     var index;
     if (opts.at !== undefined) {
-      index = opts.at;
+      index = opts.index || opts.at;
```

Comment:
> I think that there are no way to fix it for Backbone prior to 1.2.0

Falling back to the `_.indexOf` search when `opts.index === undefined` should work.

> when opts.index is 0, opts.at is 0 too...

You're right, ignore that part of my comment.